include/modem/lte_lc.h

```diff
@@ -313,6 +313,16 @@ enum lte_lc_evt_type {
 	 * event.
 	 */
 	LTE_LC_EVT_MODEM_EVENT,
+
+	/**
+	 * Information about RAI (Release Assistance Indication) configuration.
+	 *
+	 * The associated payload is the @c lte_lc_evt.rai_cfg member of type
```

Comment:
Following the other docstrings:
```suggestion
	 * The associated payload is the @c rai_cfg member of type
```